pygpu/gpuarray.pyx

```diff
@@ -1647,15 +1650,15 @@ cdef class GpuArray:
         res = <bytes>(<char *>&h)[:sizeof(h)]
         return res
 
-    def __array__(self):
+    def __array__(self, ldtype=None):
```

Comment:
Finally, as you give a default value, I think it would work with older numpy that won't pass this parameter.

Can you confirm if there is test that trigger this in libgpuarray? If so, we can merge, otherwise, I would like such test.